workshop/controllers/ui/components/themePreviewer.lua

```diff
@@ -65,5 +173,78 @@ for _,prop in pairs(themer.types) do
    y = y + 44
 end
 
+y = 0
+
+function canvasSet(size)
+   container.parent.canvasSize = size
+   container.size = size
+end
+
+function makePresetMenu()
+   preset = shared.workshop:getSettings("themeType")
+   THISISTHERESETBUTTON.visible = false
+   customUI.visible = false
+   _, _ = pcall(canvasSet, guiCoord(1, 0, 1, 0))
+   if shared.workshop:getSettings("themeType") == "custom" then
+      THISISTHERESETBUTTON.visible = true
+      customUI.visible = true
+      _, _ = pcall(canvasSet, guiCoord(1, 0, 0, 560))
```

Comment:
why are the return values assigned to _?

Reply:
during workshop loading i was getting some crashes from the code that function contains, this function is a case of, its nice, if it works, good, if not, no big deal